packages/plugin-react/src/index.ts

```diff
@@ -184,10 +200,9 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
 
       config.plugins.forEach(
         (plugin) =>
-          (plugin.name === 'react-refresh' ||
-            plugin.name === 'vite:react-jsx') &&
+          plugin.name === 'react-refresh' &&
           config.logger.warn(
-            `[@vitejs/plugin-react] This plugin conflicts with "${plugin.name}". Please remove it.`
+            `[@vitejs/react-refresh] This plugin conflicts with "${plugin.name}". Please remove it.`
```

Comment:
The part between `[` and `]` is meant to be the plugin where the warning originated from. Maybe I should just rewrite the message to be clearer about what the user should do.